### PR TITLE
Explore page functionality

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# Feat Explore page

## Added
- Explore Page
- Categories section with functionality

## Preview
scrnli_5_4_2022_4-44-12 PM

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -16,6 +16,9 @@
   --sidenav-hover: #383838;
   --card-bg: #1c1c1c;
   --card-text-color: #aaaaaa;
+  --chip-color: #373737;
+  --chip-border: rgba(255, 255, 255, 0.1);
+  --chip-hover: rgba(255, 255, 255, 0.2);
```
Comment (akash-1318) [inline src/index.css:21]: It will be good if you use hex color here as well on place of rgba.

**Review by akash-1318 — COMMENTED** [review]
Added a simple suggestion besides that your code is good with proper file structure , no changes require from my end. good work.

File: src/Pages/ExplorePage/ExplorePage.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  border: 1px solid var(--chip-border);
+  margin: 10px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+}
+/* .chip-box:hover {
+  background-color: var(--chip-hover);
+} */
```
Comment (vikrantkode) [inline src/Pages/ExplorePage/ExplorePage.css:25]: remove commented code while pushing to github

**Review by vikrantkode — APPROVED** [review]
suggested few things..rest good to merge
You can move ahead with it

File: src/Pages/ExplorePage/ExplorePage.css (hunk 2 above)
Comment (dusariudaykumar) [reply src/Pages/ExplorePage/ExplorePage.css:25]: sure will remove it

**Pull request merged**